test/steps/kubernetes.go

```diff
@@ -43,10 +41,12 @@ func (data *Data) namespaceIsDeleted() error {
 			return err
 		}
 		// wait for deletion complete
-		err = framework.WaitFor(data.Namespace, "namespace is deleted", time.Duration(2)*time.Minute, func() (bool, error) {
-			exists, err := framework.IsNamespace(data.Namespace)
-			return !exists, err
-		})
+		err = framework.WaitFor(data.Namespace, "namespace is deleted",
```

Comment:
Instead of passing the load factor every time to the `WaitFor` function, why not just get the load factor there?

Reply:
I intend to keep the waitFor utility method generic.
It could be used for other stuff not happening in Openshift ;)

Reply:
Still, you could overload it for your needs in the BDD framework.

Reply:
sure, will update on that

Reply:
changed it. wdyt ?